### ISSUE TITLE
Jade Serpent Timewalk

### ISSUE THREAD

**Issue by Kimab** [BigWigsMods/LittleWigs#824]
Hey there, got a few errors, for trash and wise mari, joining jade serpent temple in timewalking (level 57) using newest version.
Already tried disabling all other addons except bigwigs and little wigs.
for a bit less spam, i wrote the IDs with identical erros above the errors, hope that works :) anything else, just ask <3

**X = IDs:  397881, 397889, 395859, 395872, 396001, 396018, 396073, 396020, 397899, 397914**
```
1x Invalid spell ID X in the toggleOptions for module BigWigs_Bosses_Temple of the Jade Serpent Trash.
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:620: in function <Interface/AddOns/BigWigs_Core/Core.lua:553>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:641: in function `SetupOptions'
[string "@Interface/AddOns/BigWigs_Core/BossPrototype.lua"]:334: in function `Enable'
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:119: in function <Interface/AddOns/BigWigs_Core/Core.lua:117>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:130: in function <Interface/AddOns/BigWigs_Core/Core.lua:126>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:136: in function <Interface/AddOns/BigWigs_Core/Core.lua:134>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:156: in function <Interface/AddOns/BigWigs_Core/Core.lua:145>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:162: in function `v'
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:57: in function <Interface/AddOns/BigWigs_Core/Core.lua:54>
```

Opening the options of Trash brings this errors:
**X = IDs:  397881, 397889, 395859, 395872, 396001, 396018, 396073, 396020, 397899, 397914**
```
1x Invalid option X in module BigWigs_Bosses_Temple of the Jade Serpent Trash.
[string "@Interface/AddOns/BigWigs_Core/Constants.lua"]:212: in function `GetBossOptionDetails'
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:706: in function <Interface/AddOns/BigWigs_Options/Options.lua:705>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1100: in function <Interface/AddOns/BigWigs_Options/Options.lua:954>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1124: in function <Interface/AddOns/BigWigs_Options/Options.lua:1114>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-DropDownGroup.lua"]:22: in function <...AceGUI-3.0/widgets/AceGUIContainer-DropDownGroup.lua:18>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIWidget-DropDown.lua"]:432: in function <...er/Libs/AceGUI-3.0/widgets/AceGUIWidget-DropDown.lua:423>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIWidget-DropDown-Items.lua"]:351: in function <...s/AceGUI-3.0/widgets/AceGUIWidget-DropDown-Items.lua:341>
```

**X = IDs:  397881, 397889, 395859, 395872, 396001, 396018, 396073, 396020, 397899, 397914**
```
1x No spell description was returned for id X!
[string "@Interface/AddOns/BigWigs_Core/Constants.lua"]:217: in function `GetBossOptionDetails'
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:706: in function <Interface/AddOns/BigWigs_Options/Options.lua:705>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1100: in function <Interface/AddOns/BigWigs_Options/Options.lua:954>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1124: in function <Interface/AddOns/BigWigs_Options/Options.lua:1114>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-DropDownGroup.lua"]:22: in function <...AceGUI-3.0/widgets/AceGUIContainer-DropDownGroup.lua:18>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIWidget-DropDown.lua"]:432: in function <...er/Libs/AceGUI-3.0/widgets/AceGUIWidget-DropDown.lua:423>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIWidget-DropDown-Items.lua"]:351: in function <...s/AceGUI-3.0/widgets/AceGUIWidget-DropDown-Items.lua:341>
```

**X = IDs: 397785, 397797**
```
1x Invalid spell ID X in the toggleOptions for module BigWigs_Bosses_Wise Mari.
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:620: in function <Interface/AddOns/BigWigs_Core/Core.lua:553>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:641: in function `SetupOptions'
[string "@Interface/AddOns/BigWigs_Core/BossPrototype.lua"]:334: in function `Enable'
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:119: in function <Interface/AddOns/BigWigs_Core/Core.lua:117>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:130: in function <Interface/AddOns/BigWigs_Core/Core.lua:126>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:136: in function <Interface/AddOns/BigWigs_Core/Core.lua:134>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:156: in function <Interface/AddOns/BigWigs_Core/Core.lua:145>
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:162: in function `v'
[string "@Interface/AddOns/BigWigs_Core/Core.lua"]:57: in function <Interface/AddOns/BigWigs_Core/Core.lua:54>
```

Opening the options of Wise Mari brings this errors:
**X = IDs: 397785, 397797**
```
1x No spell description was returned for id X!
[string "@Interface/AddOns/BigWigs_Core/Constants.lua"]:217: in function `GetBossOptionDetails'
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:706: in function <Interface/AddOns/BigWigs_Options/Options.lua:705>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1100: in function <Interface/AddOns/BigWigs_Options/Options.lua:954>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1124: in function <Interface/AddOns/BigWigs_Options/Options.lua:1114>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-DropDownGroup.lua"]:66: in function `SetGroup'
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1180: in function <Interface/AddOns/BigWigs_Options/Options.lua:1127>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1234: in function <Interface/AddOns/BigWigs_Options/Options.lua:1229>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-TreeGroup.lua"]:508: in function `SetSelected'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-TreeGroup.lua"]:185: in function <...ibs/AceGUI-3.0/widgets/AceGUIContainer-TreeGroup.lua:181>
```

**X = IDs: 397785, 397797**
```
1x No spell description was returned for id X! 
[string "@Interface/AddOns/BigWigs_Core/Constants.lua"]:217: in function `GetBossOptionDetails'
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:706: in function <Interface/AddOns/BigWigs_Options/Options.lua:705>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1100: in function <Interface/AddOns/BigWigs_Options/Options.lua:954>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1124: in function <Interface/AddOns/BigWigs_Options/Options.lua:1114>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-DropDownGroup.lua"]:66: in function `SetGroup'
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1180: in function <Interface/AddOns/BigWigs_Options/Options.lua:1127>
[string "@Interface/AddOns/BigWigs_Options/Options.lua"]:1234: in function <Interface/AddOns/BigWigs_Options/Options.lua:1229>
[string "=[C]"]: ?
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ddOns/!KalielsTracker/Libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-TreeGroup.lua"]:508: in function `SetSelected'
[string "@Interface/AddOns/!KalielsTracker/Libs/AceGUI-3.0-41/widgets/AceGUIContainer-TreeGroup.lua"]:185: in function <...ibs/AceGUI-3.0/widgets/AceGUIContainer-TreeGroup.lua:181>
```

**Comment by Medivha** [comment]
can confirm got the same errors with the same ids and was about to report it ;)

**Comment by ntowle** [comment]
Resolved with the release of 10.0.2.
